src/runtime-rs/src/shim-proto/examples/shim-proto-ttrpc-client.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019 Ant Financial
+// Copyright (c) 2021 Ant Group
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
What *would* be useful in the banner is some indication of what this file is for.  I think it's a debugging tool?

Reply:
the `shim-proto` is designed to be published on crates.io. So these examples files are just examples for how to use  `shim-proto` crate. They are not related to kata directly